Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -325,6 +325,7 @@ def SetPlatformEnv(self):
         self.env.SetValue("CONF_AUTOGEN_INCLUDE_PATH", self.edk2path.GetAbsolutePathOnThisSystemFromEdk2RelativePath("QemuSbsaPkg", "Include"), "Platform Defined")
         self.env.SetValue("MU_SCHEMA_DIR", self.edk2path.GetAbsolutePathOnThisSystemFromEdk2RelativePath("QemuSbsaPkg", "CfgData"), "Platform Defined")
         self.env.SetValue("MU_SCHEMA_FILE_NAME", "QemuSbsaPkgCfgData.xml", "Platform Hardcoded")
+        self.env.SetValue("HAF_TFA_BUILD", "TRUE", "Default to false when running locally")

```

Comment:
Should this be false by default?. Also this description is more so to specify where it is being set from, so that if a variable is set differently then you expect, when you dump all of them, you get the description of where it was set. I would set it to "Platform Hardcoded"